docker/datanode/README.md

```diff
@@ -19,6 +19,7 @@ services:
       GRAYLOG_DATANODE_ROOT_PASSWORD_SHA2: "<root-pw-sha2>"
       GRAYLOG_DATANODE_MONGODB_URI: "mongodb://mongodb:27017/graylog"
       GRAYLOG_DATANODE_SINGLE_NODE_ONLY: "true"
+      GRAYLOG_DATANODE_CONFIG_LOCATION: "/var/lib/graylog-datanode/bin/config"
```

Comment:
I think we should set this to a default value in the entrypoint script.

And why is it `bin/config` instead of just `config`? The `bin` directories usually only contain binaries but not configuration files.

Reply:
Thanks for the feedback. @todvora is probably refactoring/replacing this config setting internally with a more appropriate default next week, so I will remove this setting here once it's done

Reply:
we changed the default, so I remove the setting here